api/main.py

```diff
@@ -202,6 +217,96 @@ async def recommend_file(request: Request, pdf: UploadFile = File(...), size: in
     return JSONResponse(content=json_compatible_item_data)
 
 
+@app.post(
+    "/recommend_text",
+    dependencies=[Depends(RateLimiter(times=rate_times, seconds=rate_seconds))],
+)
+async def recommend_text(
+    request: Request,
+    sow: Sow,
+    size: int = 10,
+    index_types=["flat", "flat_sklearn"],
+    vectorizer_types=["tf_idf"],
+):
+    """Given an input of Statement of Work as text,
+    return a JSON of recommended standards.
+    """
+    start = time.time()
+    in_text = sow.text_field
+    # df_file = "data/feather_text"
+    list_of_texts = extraction.get_list_of_text(es)
+    vectorizers, vector_storage, vector_indexes = extraction.load_into_memory(
```

Comment:
@vlall it will be an overhead to lead these into memory everytime someone requests predictions. They can be done only once when the API server starts!